rrabinovitch/Crate
# Add images for style survey

#### What’s this PR do?
- Add scary spice images for survey

#### Where should the reviewer start?
- in code/web/public/surveyImages; added images to scary folder and fixed file paths to display new images

#### How should this be manually tested?
- in terminal, run npm start. go to app in browser and add a crate subscription to see images in style survey

#### Any background context you want to provide?
- Was unable to add images at the time due to losing my laptop and being focused on getting new computer up and running

#### What are the relevant tickets?
resolves #18 resolves #19

#### Screenshots (if appropriate)

#### Questions:

--------
- [ ] I checked that the tests are running successfully before submitting this PR
